libtransmission/web.cc

```diff
@@ -592,6 +598,11 @@ class tr_web::Impl
             (void)curl_easy_setopt(e, CURLOPT_COOKIEFILE, file.c_str());
         }
 
+        if (!proxy_url.empty())
```

Comment:
Call `mediator.proxyUrl()` here instead, so that we can simplify `tr_web`'s lifecycle.